### PR TITLE
Installed Nodemon, fixed Readme and implanted timestamps on all models

### PR DESCRIPTION
**Pull request by emersonmellado**
I just did what the subject says: 

1 - Installed Nodemon;
2 - Fixed Readme 
3 - Implanted timestamps on all models

The thing is a plugin of mine just formatted all the js scripts edited, but the content is the same. I am not sure if it is a problem but it looks more readable to me.

I appreciate the acceptance of this pull request.

### CODE REVIEW

File: src/schemas/auth-schema.js [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
-	this.schema = {
-		accessToken: Joi.string(),
-		accessSecret: Joi.string()
-	};
+function AuthSchema() {
+    this.schema = {
+        accessToken: Joi.string(),
+        accessSecret: Joi.string()
+    }, {
+        timestamps: true
```
Comment (leomcabral) [inline src/schemas/auth-schema.js]: This is not a mongoose object and should not use this king "timestamps". In fact, the object holding timestamps is being discarted, as the construct variable = {}, {} wil hold only the first literal object, the second one will not be referenced by any other variable.

Obs.: the *-schema.js files are used as a schema for the REST service validators.

File: src/schemas/pickup-schema.js [hunk 2]
```diff
@@ -33,8 +31,8 @@ function PickupSchema() {
         authorization: Joi.object({
             'Authorization': Joi.string().description('Authorization Token')
         }).unknown()
+    }, {
+        timestamps: true
```
Comment (leomcabral) [inline src/schemas/pickup-schema.js]: This is not a mongoose object and should not use this king "timestamps". In fact, the object holding timestamps is being discarted, as the construct variable = {}, {} wil hold only the first literal object, the second one will not be referenced by any other variable.

Obs.: the *-schema.js files are used as a schema for the REST service validators.

File: src/schemas/user-schema.js [hunk 3]
```diff
@@ -17,29 +15,26 @@ function UserSchema() {
...
+        })),
+        social: Joi.array().items(Joi.object({
+            type: Joi.string().required().valid('facebook', 'google', 'twitter').description('Social authentication service identificator'),
+            username: Joi.string().required()
+        })),
         authorization: Joi.object({
             'Authorization': Joi.string().description('Authorization Token')
         }).unknown()
+    }, {
+        timestamps: true
```
Comment (leomcabral) [inline src/schemas/user-schema.js]: This is not a mongoose object and should not use this king "timestamps". In fact, the object holding timestamps is being discarted, as the construct variable = {}, {} wil hold only the first literal object, the second one will not be referenced by any other variable.

Obs.: the *-schema.js files are used as a schema for the REST service validators.

**Comment by leomcabral** [comment]
@EmersonMellado, take a look at the comments. Remove the "timestamps" from the *-schema.js files and redo the pull request.

**Comment by EmersonMellado** [comment]
Done. I don't thought about the validation. I expected it to be located on validation folder. LOL.

**Comment by leomcabral** [comment]
This schema thing I don't understand either, but... it was there and I'll keep it

**Pull request merged**